src/Table.luna

```diff
@@ -71,27 +85,128 @@ class Table:
+    
+    # Insert a single `Column` value to `Table` under given index.
+    # 
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" Int64Type l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > l3 = [21,22,23,24,25]
+    # > col3 = Column.fromList "col3" Int64Type l3 
+    # > newTable = table.addColumn 1 col3
+    #
+    # `index`: Integer index for newly inserted column.
+    # `column`: `Column` value to insert.
+    # `return`: `Table` value containing previous `table` with newly inserted column.
+
     def addColumn index column:
         self.fromWrapper $ self.ptr.addColumn index column.ptr
     def concat ts:
         cols = ts . each .toList . concat
         self.fromColumns cols
     def + that: Table.concat [self, that]
+
+    # Returns a `Table` value contain column matching given name.
+    #
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" Int64Type l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > column = table.at "col1"
+    #
+    # `name`: Label for column.
+    # `return`: `Table` value from the `table` for given name.
+
     def at name:
         self.columns [name]
+    
+    # Add a single `Column` value to `Table` with given name.
+    # 
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" Int64Type l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > l3 = [21,22,23,24,25]
+    # > col3 = Column.fromList "col3" Int64Type l3 
+    # > newTable = table.setAt "foo" col3
+    #
+    # `name`: String name for newly added column.
+    # `column`: `Column` value to add.
+    # `return`: `Table` value containing previous `table` with newly added column.
+
     def setAt name column:
         cols = self.toList
         before = cols.takeWhile c: (c.name == name) . not
         after = cols.drop (before.length + 1)
         col = column.rename name
         columns = before + [col] + after
         self.fromColumns columns
+
+    # Returns `Column` value from `Table` located under given name.
+    #
+    # > import Dataframes.Column 
+    # > import Dataframes.Types
+    # > import Dataframes.Table 
+    # > l1 = [1,2,3,4,5]
+    # > l2 = [11,12,13,14,15]
+    # > col1 = Column.fromList "col1" Int64Type l1
+    # > col2 = Column.fromList "col2" Int64Type l2
+    # > table = Table.fromColumns [col1 , col2]
+    # > col = table.column "col1"
+    #
+    # `name`: Name of the column to return.
```

Comment:
What if a column does not exist?

Reply:
This is a confusing choice of words, I'm afraid. Is the table located under this given name? That's how I would understand it. Maybe we could go with something like `Lookup a Column by name`? The `Table` part can be figured out from the context quite easily. This applies to the whole family of indexing/adding/removing methods.

Reply:
@piotrMocz we have very similar methods but some of them returns columns and some tables: so we have `Table.column "foo"` and it will return column `foo` and we have method `Table.at "foo"` which will give you Table with single column `foo` this is why I describe over and over returns values in a short description